recipes/expat/stdint.h

```diff
@@ -0,0 +1,259 @@
+// ISO C9x  compliant stdint.h for Microsoft Visual Studio
+// Based on ISO/IEC 9899:TC2 Committee draft (May 6, 2005) WG14/N1124
```

Comment:
It is. This recipe precedes the `msinttypes` package. I will make the change and check what happens.